ios/Flutter/Zulip.xcconfig

```diff
@@ -0,0 +1,9 @@
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+// TODO(firebase/flutterfire#13323): remove this flag
```

Comment:
I guess we've talked about having a tool to scan for TODO comments that point to closed issues, and when we do it could potentially understand this syntax for external issues. So it's fine to include it.

But let's also include an actual link. That will make it nice and easy for someone who happens across this a month from now or a year from now to quickly check on the state of the upstream issue, even if we haven't built that potential tool by then.